qtpy/QtCharts.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright © 2019- The Spyder Development Team
+#
+# Licensed under the terms of the MIT License
+# (see LICENSE.txt for details)
+# -----------------------------------------------------------------------------
+"""Provides QtChart classes and functions."""
+
+# Local imports
+from . import PYQT5, PYSIDE2, PythonQtError
+
+if PYQT5:
+    try:
+        from PyQt5 import QtChart as QtCharts
+    except ImportError:
+        raise PythonQtError('QtChart module is not installed')
```

Comment:
Please change this to

    The QtChart module was not found. It needs to be installed separately for PyQt5.

respecting, of course, the 79 width column limit we have in our codebase.